cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -153,11 +153,18 @@ function processEvent(event, callback) {
     const newState = message.detail.newEvaluationResult.complianceType;
     messageText = `${alarmName} state is now ${newState}`;
     sendMessageToSlack(messageText, callback);
-  } else if (message.source == "aws.ssm") {
+  } else if (message.source == "aws.ssm" || message.source == "aws.signin") {
     const eventName = message.detail.eventName;
-    const userName = message.detail.userIdentity.userName;
     const sourceIPAddress = message.detail.sourceIPAddress;
-    messageText = `${userName} initiated AWS Systems Manager (SSM) event ${eventName}`;
+
+    messageText = `uninitialized message text`;
+    if (message.source == "aws.ssm") {
+      const userName = message.detail.userIdentity.userName;
+      messageText = `${userName} initiated AWS Systems Manager (SSM) event ${eventName}`;
+    } else if (message.source == "aws.signin") {
+      const userType = message.detail.userIdentity.type;
```

Comment:
Interesting, this only captures root sign-ins?

Reply:
Yes, only because that is was was requested by SEAB-2233

Reply:
I was worried the other way around, that it would be too noisy and show everybody's logins. But that was before I'd seen your CF PR, where you are filtering for root only.